js/src/Ice/ObjectPrxExtensions.js

```diff
@@ -405,7 +405,7 @@ ObjectPrx._invoke = function (p, name, mode, fmt, ctx, marshalFn, unmarshalFn, u
     });
 
     try {
-        r.prepare(name, mode, ctx);
+        r.prepare(mode, ctx);
```

Comment:
Removed `name` because the implementation never uses it. This is the only call to `prepare`.